README.md

```diff
@@ -1,56 +1,52 @@
-# THE PROJECT IS ARCHIVED
+# FlakeHeaven
 
-Forks: https://github.com/orsinium/forks
-
----
+[![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
 
-# ![FlakeHell](./assets/logo.png)
+FlakeHeaven is a [Flake8](https://gitlab.com/pycqa/flake8) wrapper to make it cool.
 
-[![PyPI version](https://badge.fury.io/py/flakehell.svg)](https://badge.fury.io/py/flakehell)
-[![Build Status](https://cloud.drone.io/api/badges/life4/flakehell/status.svg)](https://cloud.drone.io/life4/flakehell)
-[![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
-[![Documentation](https://readthedocs.org/projects/flakehell/badge/?version=latest)](https://flakehell.readthedocs.io/)
+This project is a fork of [FlakeHell](https://github.com/life4/flakehell). FlakeHell and the official fork of it at
```

Comment:
I'd add some note that this is now the new official fork maintained by the community, maybe pointing to the issue of flakehell where we decided to give the step forward

Reply:
what do you mean by official? I mean, it's as official as any other fork!

Reply:
Maybe official is not the correct wording, what I meant is that this is the fork that is going to be maintained by the community of all the existent forks, and that the previous maintainer agreed to do this migration.

I'd find it interesting from a user stand point, as right now there are a lot of forks and tools with similar names that do the same. It will help them choose